orchagent/pfcwdorch.cpp

```diff
@@ -272,7 +272,12 @@ task_process_status PfcWdOrch<DropHandler, ForwardHandler>::deleteEntry(const st
     SWSS_LOG_ENTER();
 
     Port port;
-    gPortsOrch->getPort(name, port);
+
+    if(!gPortsOrch->getPort(name, port))
+    {
+        SWSS_LOG_WARN("Failed to stop PFC Watchdog port not found %s", name.c_str());
+        return task_process_status::task_failed;
```

Comment:
Though the processing remains the same, better return code would be task_process_status::task_invalid_entry